src/shared/components/palette/complex/complex.module.ts

```diff
@@ -24,7 +25,8 @@ import { CcdCYAPageLabelFilterPipe } from './ccd-cyapage-label-filter.pipe';
     CaseLinkModule,
     BaseFieldModule,
     PaletteUtilsModule,
-    ConditionalShowModule
+    ConditionalShowModule,
+    CaseFlagModule
```

Comment:
@johnbenjamin-hmcts - is it intended for `CaseFlagModule` to be used within `ComplexModule`?

Reply:
Thank you @Daniel-Lam , this was no longer required and I have now updated the code.